webui/react/src/hooks/useInterval.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useRef } from 'react';
+
+const useInterval = (callback: () => void, delay: number): void => {
```

Comment:
**suggestion:** we do have a `usePolling` hook. It's actually not accurately named anymore. It's more accurate to say that it is running a function on an intervals, but with more bells and whistles. It can take an async function and take in an option to wait for it to resolve before calling the next. Think it might be worth it to use that hook instead?

Reply:
will do